openIMIS/developer_tools/skeletons/calculation_rule.py.template

```diff
@@ -21,12 +21,12 @@ class {{class_name}}(AbsCalculationRule):
     type = "type"
     sub_type = "sub_type"
 
-    ##signal_get_rule_name = Signal(providing_args=[])
```

Comment:
I am really surprise that this was still there: it is not compatible with Django 4 ...